cms/static/js/views/course_video_settings.js

```diff
@@ -93,12 +95,23 @@ function($, Backbone, _, gettext, moment, HtmlUtils, StringUtils, TranscriptSett
             return fidelityPlan;
         },
 
-        getPlanLanguages: function() {
-            var selectedPlan = this.availableTranscriptionPlans[this.selectedProvider];
-            if (this.selectedProvider === CIELO24) {
-                return selectedPlan.fidelity[this.selectedFidelityPlan].languages;
+        getSourceTranslations: function() {
+            var sourceTranslations = null;
+            if (this.selectedProvider === THREE_PLAY_MEDIA) {
```

Comment:
I don't think we need this if condition because this method only works for 3playmedia.